app/scripts/controllers/locationPickerMap.ctrl.js

```diff
@@ -71,19 +65,21 @@ angular.module('yourStlCourts').controller('LocationPickerMapCtrl', function ($s
   }
 
   leafletData.getMap("municipalityMap").then(function(map){
-    geoJson = L.geoJSON(countyMunicipalityBoundaryData,{
-      style: function(feature) {
-        return {
-          fillColor: "",
-          weight: 1,
-          opacity: 1,
-          color: 'blue',
-          dashArray: '',
-          fillOpacity: 0
-        }
-      },
-      onEachFeature: onEachFeature
-    }).addTo(map);
+    $http.get('data/stlCountyMunicipalBoundaries.json').success(function(geoJsonData){
```

Comment:
Try to always use `promise.then` instead of angular's `promise.success`. It's more consistent with other promise usages